<?xml version="1.0"?>
<launch>

	<param name="pose_topic" value="/mavros/local_position/pose" />
	<param name="yaml_file" value="/config/mpc.yaml" />

	<node pkg="airo_control" type="system_identification" name="system_identification_node" output="screen">
		<rosparam command="load" file="$(find airo_control)/config/mpc.yaml" />
	</node>

</launch>